.github/workflows/unittests.yml

```
@@ -75,9 +75,9 @@ jobs:
       - name: Install pip dependencies
         run: pip install tensorflow==2.1.0 larq~=0.9.4 larq_zoo==1.0.0 pytest tensorflow_datasets==3.1.0 --no-cache-dir
       - name: Run FileCheck tests
-        run: bazelisk test larq_compute_engine/mlir/tests:all --test_output=all --distinct_host_configuration=false $([ -z "$GOOGLE_APPLICATION_CREDENTIALS" ] || echo "--remote_http_cache=https://storage.googleapis.com/plumerai-bazel-cache/lce-ubuntu --google_default_credentials")
+        run: bazelisk test larq_compute_engine/mlir/tests:all --test_output=all --copt=-O2 --distinct_host_configuration=false $([ -z "$GOOGLE_APPLICATION_CREDENTIALS" ] || echo "--remote_http_cache=https://storage.googleapis.com/plumerai-bazel-cache/lce-ubuntu --google_default_credentials")
```

```suggestion
        run: bazelisk test larq_compute_engine/mlir/tests:all --test_output=all --distinct_host_configuration=false $([ -z "$GOOGLE_APPLICATION_CREDENTIALS" ] || echo "--remote_http_cache=https://storage.googleapis.com/plumerai-bazel-cache/lce-ubuntu --google_default_credentials")
```

In the last commit that failed CI before you added in the `--copt=-02` (f8ff469) the filecheck test passed and it was just the end-to-end test that failed, so would you be okay to revert this for the filecheck test?

---

>In the last commit that failed CI before you added in the --copt=-02 (f8ff469) the filecheck test passed and it was just the end-to-end test that failed, so would you be okay to revert this for the filecheck test?

If I understand bazel caching correclty, we shouldn't do this.
If we revert this flag in the filecheck test the local cache won't be shared between the two commands so the end2end tests would need to rebuild parts of the code that was already compiled for the filecheck tests since `--copt` changed.

---

> If I understand bazel caching correclty, we shouldn't do this.
> If we revert this flag in the filecheck test the local cache won't be shared between the two commands so the end2end tests would need to rebuild parts of the code that was already compiled for the filecheck tests since `--copt` changed.

Ah I see, fair enough